src/main/java/com/eternalcode/discordapp/command/MinecraftServerInfoCommand.java

```diff
@@ -0,0 +1,106 @@
+package com.eternalcode.discordapp.command;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.jagrosh.jdautilities.command.SlashCommand;
+import com.jagrosh.jdautilities.command.SlashCommandEvent;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+
+import java.awt.*;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.time.Instant;
+import java.util.List;
+
+public class MinecraftServerInfoCommand extends SlashCommand {
+
+    private static final String API_URL = "https://api.mcsrvstat.us/2/";
+    private static final String IMAGE_API_URL = "https://api.mcsrvstat.us/icon/%s";
+
+    public MinecraftServerInfoCommand() {
+        this.name = "minecraft";
+        this.aliases = new String[]{ "mc", "mcserver" };
+        this.options = List.of(
+                new OptionData(OptionType.STRING, "address", "The domain/IP address of the Minecraft server")
+                        .setRequired(true)
+        );
+    }
+
+    @Override
+    public void execute(SlashCommandEvent event) {
+        String domain = event.getOption("domain").getAsString();
+
+
+        this.sendApiRequest(domain);
+
+        JsonObject response = JsonParser.parseString(this.sendApiRequest(domain)).getAsJsonObject();
```

Comment:
upo chyba, nie po co wysyłasz dwa razy tą metodę `sendApiRequest`? Proponuję zrobić to tak:

```
String request = this.sendApiRequest(domain);

JsonObject response = JsonParser.parseString(request).getAsJsonObject();
```